osmaxx/excerptexport/static/excerptexport/scripts/maps/excerpt_viewer.js

```diff
@@ -1,8 +1,22 @@
-window.ExcerptViewer = function(mapElementID, excerptApiUrl) {
+window.ExcerptViewer = function(mapElementID, excerptApiUrl, disableZoom) {
```

Comment:
after a short discussion: using a function that can be called should solve this issue quite neatly, doing that now.